resources/default-config.edn

```diff
@@ -1,4 +1,4 @@
-{:osm-url "resources/osm/saarland.osm.bz2"
+{:osm-url "https://github.com/hiposfer/kamal/raw/master/resources/osm/saarland.osm.bz2"
```

Comment:
Won't this cause the file to be downloaded on every fresh start?

Reply:
I am actually not sure about it. I dont know how does the cache works outside of the browser. I hope that it is more like "download once, reuse afterwards"

Reply:
@mehdisadeghi I created a ticket for this #64. Since we dont know if files are being cached or not. I think further discussion wont help much here. We would need to address the problem properly at some point. I think that for the time being it is ok to just move on.

Reply:
oh wait. This makes no sense !! If the file is being ignored from the `uberjar` then there is no need to actually rely on the remote file. We can just take the local one :D

Reply:
as you said, in production the file should be configured so this is only `dev` relevant, so it can perfectly be local.

Reply:
@carocad Yes it makes sense. This is all dev related, so it's OK.